contracts/DissentVoting.sol

```diff
@@ -0,0 +1,388 @@
+    * @notice Change dissent window to  `_dissentWindow` seconds??
+    * @param _minAcceptQuorumPct New dissent window
+    */
+    function changeDissentWindow(uint64 _dissentWindow)
+        external
+        authP(MODIFY_DISSENT_WINDOW, arr(uint256(_dissentWindow), uint256(dissentWindow)))
+    {
+        dissentWindow = _dissentWindow;
+
+        emit ChangeDissentWindow(_dissentWindow);
+    }
+
+    /**
+    * @notice Create a new vote about "`_metadata`"
+    * @param _executionScript EVM script to be executed on approval
+    * @param _metadata Vote metadata
+    * @return voteId Id for newly created vote
+    */
+    function newVote(bytes _executionScript, string _metadata) external auth(CREATE_VOTES_ROLE) returns (uint256 voteId) {
+        return _newVote(_executionScript, _metadata, true, true);
+    }
+
+    /**
+    * @notice Create a new vote about "`_metadata`"
+    * @param _executionScript EVM script to be executed on approval
+    * @param _metadata Vote metadata
+    * @param _castVote Whether to also cast newly created vote
+    * @param _executesIfDecided Whether to also immediately execute newly created vote if decided
+    * @return voteId id for newly created vote
+    */
+    function newVote(bytes _executionScript, string _metadata, bool _castVote, bool _executesIfDecided)
+        external
+        auth(CREATE_VOTES_ROLE)
+        returns (uint256 voteId)
+    {
+        return _newVote(_executionScript, _metadata, _castVote, _executesIfDecided);
+    }
+
+    /**
+    * @notice Vote `_supports ? 'yea' : 'nay'` in vote #`_voteId`
+    * @dev Initialization check is implicitly provided by `voteExists()` as new votes can only be
+    *      created via `newVote(),` which requires initialization
+    * @param _voteId Id for vote
+    * @param _supports Whether voter supports the vote
+    * @param _executesIfDecided Whether the vote should execute its action if it becomes decided
+    */
+    function vote(uint256 _voteId, bool _supports, bool _executesIfDecided) external voteExists(_voteId) {
+        require(canVote(_voteId, msg.sender), ERROR_CAN_NOT_VOTE);
+        _vote(_voteId, _supports, msg.sender, _executesIfDecided);
+    }
+
+    /**
+    * @notice Execute vote #`_voteId`
+    * @dev Initialization check is implicitly provided by `voteExists()` as new votes can only be
+    *      created via `newVote(),` which requires initialization
+    * @param _voteId Id for vote
+    */
+    function executeVote(uint256 _voteId) external voteExists(_voteId) {
+        require(canExecute(_voteId), ERROR_CAN_NOT_EXECUTE);
+        _executeVote(_voteId);
+    }
+
+
+    /**
+    * @notice ACLOracle that indicates if `who` can perform certain action based on most recent yes vote
+    * @dev IACLOracle interface conformance
+    */    
+    function canPerform(address who, address where, bytes32 what, uint256[] how) external view returns (bool) {
+        return lastYeaVote[who] < getTimestamp64().sub(dissentWindow);
```

Comment:
If we want to also be able to pass the `DissentWindow` through an `authP(SOME_ROLE, arr(dissentWindow))` on a function then we would check the first element in `how[]` for a value, and if present and formatted as expected, use that value, otherwise use the `dissentWindow` specified in the DissentVoting contract.

Reply:
Forgot to add that one.